magefile.go

```diff
@@ -1749,10 +1752,10 @@ func createTestRunner(matrix bool, singleTest string, goTestFlags string, batche
 	fmt.Printf(">>>> Using %s instance provisioner\n", instanceProvisionerMode)
 	stackProvisionerMode := os.Getenv("STACK_PROVISIONER")
 	if stackProvisionerMode == "" {
-		stackProvisionerMode = "ess"
+		stackProvisionerMode = "stateful"
 	}
-	if stackProvisionerMode != "ess" && stackProvisionerMode != "serverless" {
-		return nil, errors.New("STACK_PROVISIONER environment variable must be one of 'serverless' or 'ess'")
+	if stackProvisionerMode != "stateful" && stackProvisionerMode != "serverless" {
+		return nil, errors.New("STACK_PROVISIONER environment variable must be one of 'serverless' or 'stateful'")
```

Comment:
It would be nice to also return the value we got, something like:
```suggestion
		return nil, errors.New("STACK_PROVISIONER environment variable must be one of 'serverless' or 'stateful', not %q", stackProvisionerMode)
```

We can probably find a better wording, but you get the idea.